lib/stripe_mock/request_handlers/helpers/subscription_helpers.rb

```diff
@@ -11,7 +11,7 @@ def resolve_subscription_changes(subscription, plans, customer, options = {})
         subscription[:items][:data] = plans.map do |plan|
           if options[:items] && options[:items].size == plans.size
             quantity = options[:items] &&
-              options[:items].values.detect { |item| item[:plan] == plan[:id] }[:quantity] || 1
+              options[:items].detect { |item| item[:plan] == plan[:id] }[:quantity] || 1
```

Comment:
FYI this is the new structure for the `items` attribute. See: https://stripe.com/docs/api#create_subscription